.awestruct_ignore

```diff
@@ -13,3 +13,6 @@ add-your-event.html.slim
 stylesheets/fonts/SIL Open Font License.txt
 stylesheets/rh-icons
 CONTRIBUTING.md
+_docker
+_cucumber
+cucumber_failures
```

Comment:
`cucumber_failures` needs to be added, but `_docker` and `_cucumber` don't need to be. They start with `_` which is auto-ignored by awestruct.